cfDNApipe/cfDNA_utils.py

```diff
@@ -536,24 +574,32 @@ def ComputeOCF(bedgz, txtOutput, OCFOutput, regionFile):
             #                if i >= region_Start and i <= region_End:
             #                    covPOS[i][0] += 1
 
-            if rstart >= region_Start and rstart <= region_End:  # consider read start point, U end
+            if (
+                rstart >= region_Start and rstart <= region_End
+            ):  # consider read start point, U end
                 covPOS[rstart][1] += 1
 
-            if rend >= region_Start and rend <= region_End:  # consider read start point, D end
+            if (
+                rend >= region_Start and rend <= region_End
+            ):  # consider read start point, D end
                 covPOS[rend][2] += 1
 
         # finished a region
         midpoint = int((region_Start + region_End) / 2)
 
-        Udata = np.vstack((Udata, [covPOS[x][1] for x in range(midpoint - 120, midpoint + 121)]))
-        Ddata = np.vstack((Ddata, [covPOS[x][2] for x in range(midpoint - 120, midpoint + 121)]))
-
-        left_OCF = sum([covPOS[x][2] for x in range(midpoint - 70, midpoint - 50)]) - sum(
-            [covPOS[x][1] for x in range(midpoint - 70, midpoint - 50)]
+        Udata = np.vstack(
+            (Udata, [covPOS[x][1] for x in range(midpoint - 120, midpoint + 121)])
```

Comment:
Didn't you remove Udata and Ddata in lines #548 and 549? Where is the initialization for these variables?

Reply:
These are just formatting changes.